src/webgl/material.js

```diff
@@ -313,6 +330,9 @@ p5.prototype.createFilterShader = function(fragSrc) {
  * Sets the <a href="#/p5.Shader">p5.Shader</a> object to
  * be used to render subsequent shapes.
  *
+ * Shaders can alter the positioning of shapes drawn with them.
+ * To ensure consistency in rendering, it's recommended to use the vertex shader in the <a href="#/p5/createShader">createShader example</a>
```

Comment:
Just adding a period to the end of the sentence 🙂 
```suggestion
 * To ensure consistency in rendering, it's recommended to use the vertex shader in the <a href="#/p5/createShader">createShader example</a>.
```